tests/tests/Features/service-container.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Entity Param Converter
```

Comment:
I think, you can merge there two scenarios (serviceContainer and  serviceModel) in one file.